Automattic/woocommerce-payments
# Bump WC tested up to and WC minimum version for 4.7.0 release

Fixes #

#### Changes proposed in this Pull Request

- Bump minimum WC version from 6.2.2 to 6.4.1
- Add WC 6.9.0 (beta to be released on Tuesday 23) as the "WC tested up to" version

No WP updates to catch up on.

#### Testing instructions

- Verify that wherever we used 6.2 as a minimum before, we now use 6.4. Update. Workflows to be updated in a separate PR as per [this conversation](https://github.com/Automattic/woocommerce-payments/pull/4615#discussion_r951952678).
- Please check the CI tests in the GH workflows.

-------------------


**Post merge**
